relentless/optimize/objective.py

```diff
@@ -6,8 +6,7 @@
 by adjusting the variables on which the function depends.
 
 This function, :math:`f`, is a scalar value that is defined as a function of :math:`n`
-problem :class:`DesignVariables<~relentless.variable.DesignVariable>`
-:math:`\mathbf{x}=\left[x_1,\ldots,x_n\right]`.
+problem :class:`~relentless.variable.DesignVariable`s :math:`\mathbf{x}=\left[x_1,\ldots,x_n\right]`.
```

Comment:
I think you need to escape the s with a backslash to get this to set correctly
```suggestion
problem :class:`~relentless.variable.DesignVariable`\s :math:`\mathbf{x}=\left[x_1,\ldots,x_n\right]`.
```